### PR TITLE
Ais can no longer be on fire

### LINKED ISSUE #9156: AI can be set on fire and burn indefinitely and there's no fire overlay
**Problem Description**:
AI can be set on fire (for instance in an explosion) and burn indefinitely with no visual feedback on the outside. People (and borgs) can be set on fire by it and it also might make pulling by normal crew hard or impossible. Fire does not harm AI, so what's the point?

**What did you expect to happen**: 
Not to be set on fire, the fire to fade out or the fire overlay to display on top of the core.

**What happened instead**:
Got set on fire indefinitely with no visual feedback for others.

**Why is this bad/What are the consequences:**
AI does not get harmed by the fire (so AI becomes a living torch for no good reason), there's no fire overlay, it's generally confusing.

**Steps to reproduce the problem**:
1. Get set on fire as AI.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**:  Caused by an explosion.

### PR DESCRIPTION
**Pull request by alexkar598**
Fixes #9156 

Yarr

### CODE REVIEW

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 1]
```diff
@@ -1239,4 +1239,16 @@ var/list/ai_verbs_default = list(
...
+		to_chat(src, "<span class='warning'>Target is not on or near any active cameras on the station.</span>")
+		
+/mob/living/silicon/ai/handle_fire()
+	return
+
+/mob/living/silicon/ai/update_fire()
+	return
+
+/mob/living/silicon/ai/IgniteMob()
+	return 0
```
Comment (KasparoVy) [inline code/modules/mob/living/silicon/ai/ai.dm]: Obligatory `return FALSE`

Comment (Crazylemon64) [reply code/modules/mob/living/silicon/ai/ai.dm]: Indeed, `return FALSE`

Comment (alexkar598) [reply code/modules/mob/living/silicon/ai/ai.dm]: lemme fix simple animals too then

**Comment by Plotron** [comment]
This 'balance' tag would only affect people trying to touch the AI, I think.

Because AI itself is unharmed in the fire.

**Comment by taukausanake** [comment]
I can imagine it now. The whole AI sat is on fire. The floor is lava. The walls are melting. The very air is literal plasma, not that purple shit the station mines, electronics exploding all over, and the AI just sitting there in the center of it all
AI: "This is fine"

**Comment by alexkar598** [comment]
I am not even touching sprite code with a 3 meter pole,no way im making fire overlays work for ai

**Pull request merged**